### ISSUE TITLE
Is there any corner case for the generation of an abnormal manifest file by DynamicTraceEventParser.WriteAllManifests?

### ISSUE THREAD

**Issue by jiahuizheng1114** [microsoft/perfview#1762]
There is an abnormal manifest file named ".manifest.xml" and the size is 0.
![image](https://user-images.githubusercontent.com/120216048/206779754-a248c5de-2272-4614-bccd-1632a57b9511.png)
Could it be caused by an abnormal DynamicProvider whose name is empty?
![image](https://user-images.githubusercontent.com/120216048/206780322-088aab7d-a9b4-4887-8f70-e1c7f99c8704.png)
Does anyone have more ideas about this issue?

**Comment by brianrob** [comment]
Yes, this does seem possible.  However this would mean that there is a provider that does not have a name.

**Comment by brianrob** [comment]
Apologies for the very late reply.  I'm going to go ahead and close this - feel free to reach out if you have further questions on this.
